### ISSUE TITLE
Uncaught Error: Class 'PhpMqtt\Client\MqttClient' not found

### ISSUE THREAD

**Issue by tukusejssirs** [php-mqtt/client-examples#3]
I installed `php-mqtt` using `composer`, but I could not load the class.

```php
require 'vendor/autoload.php';
use \PhpMqtt\Client\MqttClient;

$server   = 'localhost';
$port     = 1883;
$mqtt = new \PhpMqtt\Client\MqttClient($server, $port);
```

It always throws this error: `PHP Fatal error:  Uncaught Error: Class 'PhpMqtt\Client\MqttClient' not found`.

Is there anything else needed to make it work.

Note that I use CentOS 8 server.

**Comment by Namoshek** [comment]
Sounds like the autoloading doesn't work correctly. You are also using a wrong import (leading `\`) and an FQCN at the same time. Maybe remove the `use` and try again, after running `composer dump-autoload`.

**Comment by tukusejssirs** [comment]
I removed the `use` command, run `compoer dump-autoload`, but it still does not work.

The code I actually used:

```php
<?php
require 'vendor/autoload.php';
//use PhpMqtt\Client\MqttClient;

$server   = 'localhost';
$port     = 1883;
$mqtt = new \PhpMqtt\Client\MqttClient($server, $port);
?>
```

[Here](https://github.com/php-mqtt/client-examples/files/7121721/php_mqtt_composer.zip) are compressed all the files/folders generated by `composer` after `composer require php-mqtt/client` and `composer dump-autoload` in an empty folder.

---

FYI: I use `composer` version `2.1.6 2021-08-19 17:11:08`; `php` version `7.2.24`.
